NAMESPACE

```diff
@@ -53,6 +53,9 @@ export(getTwitter)
 export(getTwitterToken)
 export(get_sentiment)
 export(get_stopwords)
+export(get_url_param)
+export(get_url_subdomain)
+export(get_url_suffix)
```

Comment:
URL suffix could mean different things like this one.
https://survey.qut.edu.au/WebHelp/URLSuffix.htm

Top-Level Domain would be most unambiguous word here.
I know it's a little long but how about get_url_top_level_domain ?